content/docs/configuration.md

```diff
@@ -83,7 +83,30 @@ synced_files:
     dest: .
 ```
 
+## RPM spec file not in upstream repository
```

Comment:
I debate if we want to have this here or create a new document: best practices or Q&A

Reply:
We already have https://packit.dev/packit-as-a-service/#faq
It could be placed there.

Reply:
but that's faq for the service, this is a generic best practice

on the flip side, one doc can serve for both (which may actually be a good idea so that people are not confused)

Reply:
I have created a new section called FAQ https://github.com/packit-service/packit.dev/pull/36/commits/e877d65fe10018eca4d19bf6d2096d3a0bad3d72